validation/src/main/scala/hmda/validation/dsl/PredicateRegEx.scala

```diff
@@ -4,8 +4,20 @@ import scala.language.implicitConversions
 import scala.util.matching.Regex
 
 object PredicateRegEx {
+  implicit def validCensusTractFormat: Predicate[String] = new Predicate[String] {
+
+    override def validate: (String) => Boolean = {
+      val censusTractRegEx = "^[0-9]{4}.[0-9]{2}$"
```

Comment:
`\d` is a shortcut for `[0-9]`.  Actually, with the additional backslash for the escape character (so, `\\d`), it's only two characters shorter, but I think that will better fit the pattern of our regular expressions.

Also, depending on who gets their PR merged in first, we'll need tests for this regex.